index.html

```diff
@@ -40,11 +39,158 @@ <h3 class = "getintouch">
+    <div class="proj-lan">
+      <span class="lan">HTML</span>
+      <span class="lan"> CSS</span>
+      <span class="lan">Javascript</span>
+    </div>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+  <div class="second-sec-div">
+    <div>
+      <img class="proj-img" src="images/Snapshoot Portfolio (3).png" alt="picture" >
+    </div>
+    <div>
+      <strong class="proj-title">Multi-Post <br>
+        Stories</strong>
+    </div>
+    <div class="proj-intro">
+      <span class="intro-1">CANOPY</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-2">Back end Dev</span>
+      <img class="dot" src="images/Counter.png" alt="dot">
+      <span class="intro-3">2015</span>
+    </div>
+    <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <div class="proj-lan">
+      <span class="lan">HTML</span>
+      <span class="lan"> CSS</span>
+      <span class="lan">Javascript</span>
+    </div>
+    <div class="btn-div">
+      <button class="btn" type = "submit"> See Project</button>
+    </div>
+  </div>
+</section>
+<section>
+  <div class="third-section" >
+    <div>
+    <h1 class="title">
+      About Myself
+    </h1>
+    <p class="intro">
+      I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <h3 class = "getintouch">
+      LET'S CONNECT
+    </h3>
+    </div>
+    <ul class="social-media-icon" >
+     <li><a class = "social-media" href="#"><img src="images/github.png" alt="github"></a></li>
+     <li> <a class = "social-media"href="#"><img src="./images/linkedin.png" alt="linkedin"></a> </li>
+     <li><a class = "social-media"href="#"><img src="images/anglelist.png" alt="anglelist"></a> </li>
+     <li><a class = "social-media" href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+     <li><a class = "social-media"href="#"><img src="images/medium.png"alt="medium"></a></li>
+ </ul>
+  </div>
+<div class="middle-sec">
+  <div class="btn-divb">
+    <button class= "btn" type="submit">Get my resume</button>
+  </div>
+  <div>
+      <div class="chev">
+      <h3 class="p-lan">Languages </h3> 
+      <i class="fa fa-chevron-down"></i>
+      </div>
+    <div class="img">
+    <img class="lan-g" src="images/Frame 9.png" alt="Javascript">
+    <img class="lan-g" src="images/Frame 10.png" alt=" HTML">
+    <img class="lan-g" src="images/Frame 11.png" alt="CSS">
+    </div>
+  <div class="chev">
+    <h3 class="skae">Frameworks </h3>
+    <i class="fa fa-chevron-right"></i>
+    </div>
+    <hr class="line">
+    <div class="chev">
+    <h3 class="s">Skills</h3>
+    <i class="fa fa-chevron-right"></i>
+    </div>
+    <hr class="line">
   </div>
```

Comment:
- [ ] kindly consider giving this section the exact padding and margins as given in the design 
- [ ]  for the container kindly consider giving it some margin top and top-right-radius to meet the given design check the below images to understand this more clearly 
- your design 👇 
<img width="399" alt="Screen Shot 2023-03-31 at 08 58 35" src="https://user-images.githubusercontent.com/60146030/229046777-7ed093e2-5fa1-4f02-b8a7-ed3222f0f789.png">

- provided design 👇 
<img width="446" alt="Screen Shot 2023-03-31 at 09 01 39" src="https://user-images.githubusercontent.com/60146030/229048329-272df15b-f2fe-4a6a-8dc9-66a9819d653a.png">
  

all the above changes are there to make your design on a good standard and make a perfect duplicate of the provided one